code/modules/reqs/supply.dm

```diff
@@ -193,41 +199,15 @@ GLOBAL_LIST_EMPTY_TYPED(exports_types, /datum/supply_export)
 		SSpoints.shoppinglist -= "[SO.id]"
 		SSpoints.shopping_history += SO
 
-/datum/export_report
-	var/id
-	var/points
-	var/list/exports
 
 /obj/docking_port/mobile/supply/proc/sell()
-	var/list/exports = list()
-	var/points = 0
 	for(var/place in shuttle_areas)
 		var/area/shuttle/shuttle_area = place
 		for(var/atom/movable/AM in shuttle_area)
 			if(AM.anchored)
 				continue
-			var/atom/movable/atom_type = AM.type
-			if(isxeno(AM)) // deal with admin spawned upgrades
-				var/mob/living/carbon/xenomorph/X = AM
-				atom_type = X.xeno_caste.caste_type_path
-			if(GLOB.exports_types[atom_type])
-				var/datum/supply_export/E = GLOB.exports_types[atom_type]
-				if(!exports[atom_type])
-					exports[atom_type] = 0
-				exports[atom_type]++
-				points += E.cost
+			AM.supply_export(faction)
```

Comment:
Before this PR, there was two ways of selling stuff: Usin the supply export datum (there was one for each xeno caste....)
, or using the proc supply_export.

Both were doing the same thing, so i'm just using the proc now